### PR TITLE
Harmony(Text): inherit color and font-size by default

### LINKED ISSUE #856: Harmony(Text): inherit color and font-size by default
https://github.com/taskany-inc/bricks/pull/839/files#diff-754b0327a0d79a46221b9523957b98d5c56060b3bd596506da0e10ecc55bd464R31

### PR DESCRIPTION
**Pull request by Katrin-kudryash**
## PR includes

- [x] Feature

## Related issues

Resolve #856

### CODE REVIEW

File: src/harmony/Text/Text.tsx [hunk 1]
```diff
@@ -84,7 +84,7 @@ export const Text = React.forwardRef(function <T extends keyof AllowedHTMLElemen
         wordWrap,
         lines,
         ellipsis,
-        color,
+        color = 'inherit',
```
Comment (awinogradov) [inline src/harmony/Text/Text.tsx:87]: No way

**Review by awinogradov — CHANGES_REQUESTED** [review]
I'm not sure inline styles by default is the best idea

File: src/harmony/Text/Text.module.css [hunk 2]
```diff
@@ -1,7 +1,4 @@
-.Text {
-    font-size: 1rem;
-    color: var(--text-primary);
-}
+.Text {}
```
Comment (tsumo) [inline src/harmony/Text/Text.module.css:1]: No reason to keep an empty class

Comment (tsumo) [inline src/harmony/Text/Text.module.css:1]: Also, if color is removed from the base component it should be set somewhere at the root of the app

**Review by tsumo — CHANGES_REQUESTED** [review]

File: src/harmony/Text/Text.module.css (hunk 2 above)
Comment (tsumo) [reply src/harmony/Text/Text.module.css:1]: I think. Maybe it should be done in project, not here

File: src/harmony/Text/Text.stories.tsx [hunk 3]
```diff
@@ -12,7 +12,6 @@ export default meta;
 export const Default: StoryObj<typeof Text> = {
     args: {
         children: 'Simple text',
-        size: 'xs',
```
Comment (tsumo) [inline src/harmony/Text/Text.stories.tsx:15]: This is fine, keep it

File: src/harmony/Text/Text.tsx [hunk 4]
```diff
@@ -120,7 +120,7 @@ export const Text = React.forwardRef(function <T extends keyof AllowedHTMLElemen
         <Tag
             {...rest}
             ref={ref}
-            className={cn(s.Text, textStyles, className, {
+            className={cn(textStyles, className, {
```
Comment (tsumo) [inline src/harmony/Text/Text.tsx:123]: Leave main class, you need it for selectors to work

File: src/harmony/Text/Text.tsx [hunk 5]
```diff
@@ -65,8 +65,8 @@ interface AllowedHTMLElements {
 type HeadingTagName = Extract<keyof AllowedHTMLElements, `h${number}`>;
 const isHeading = (tag: string): tag is HeadingTagName => /^(h[1-6])/.test(tag);
 
-const calcTextSizes = (tag: string, size: keyof typeof textSize, weight: keyof typeof textWeight = 'regular') =>
-    isHeading(tag) ? headingClasses[tag] : [textSize[size], textWeight[weight]];
+const calcTextSizes = (tag: string, weight: keyof typeof textWeight = 'regular') =>
```
Comment (tsumo) [inline src/harmony/Text/Text.tsx:68]: Cannot just delete it

**Review by tsumo — CHANGES_REQUESTED** [review]

**Review by tsumo — APPROVED** [review]

File: src/harmony/Alert/Alert.tsx [hunk 6]
```diff
@@ -28,7 +28,7 @@ export const Alert: React.FC<AlertProps> = ({ text, icon, className, view, ...pr
     return (
         <div className={cn(s.Alert, viewMap[view], className)} {...props}>
             {nullable(icon, (i) => i, iconMap[view])}
-            <Text color="inherit">{text}</Text>
+            <Text>{text}</Text>
```
Comment (awinogradov) [inline src/harmony/Alert/Alert.tsx:31]: Does we really need Text component here now?

**Review by awinogradov — CHANGES_REQUESTED** [review]

File: src/harmony/Alert/Alert.tsx (hunk 6 above)
Comment (Katrin-kudryash) [reply src/harmony/Alert/Alert.tsx:31]: no

**Review by awinogradov — APPROVED** [review]

**Pull request merged**